app/javascript/react/components/ReviewsContainer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import ReviewTile from './ReviewTile'
+
+const ReviewsContainer = props => {
+  let reviewArray
+  if (props.reviews.length > 0) {
+    reviewArray = props.reviews.map((review) => {
+      return (
+        <ReviewTile
+          rating={review.rating}
+          comment={review.comment}
+        />
+      )})
```

Comment:
an extra line here might make this more readable, but Ill leave that up to your disgression